hiv-benchmarking/README.md

```diff
@@ -0,0 +1,87 @@
+## HIV benchmarking
+
+This folder contains raw data, processed data, and SPRAS results pertaining to the dataset taken from the following research article: **[HIV-1 virological synapse formation enhances infection spread by dysregulating Aurora Kinase B](https://doi.org/10.1371/journal.ppat.1011492)** - Bruce JW, Park E, Magnano C, Horswill M, Richards A, Potts G, et al. (2023)
+
+The study examines human immune cells responding to viral infection as well as the changes that take place inside the already infected cells, which is the focus here.
+The data is from protein abundance and phosphorylation experiments, which will be the input to pathway reconstruction.
+
+**Overarching goal:** Recreate published biological case study on HIV data using SPRAS. This will help in identifying nodes i.e. proteins that are relevant to the disease.
+
```

Comment:
Somewhere in the readme note that `requirements.txt` pins the versions of the important packages for data analysis but this assumes the Jupyter notebook environment is already available.